core/server/api/v2/utils/serializers/output/db.js

```diff
@@ -0,0 +1,41 @@
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:db');
+
+module.exports = {
+    backupContent(filename, apiConfig, frame) {
+        debug('backupContent');
+
+        frame.response = {
+            db: [{filename: filename}]
+        };
+    },
+
+    exportContent([exportedData, fileName], apiConfig, frame) {
+        debug('exportContent');
+
+        frame.response = {
+            filename: fileName,
+            db: [exportedData]
+        };
+    },
+
+    importContent(response, apiConfig, frame) {
+        debug('exportContent');
+
+        // NOTE: response can contain 2 objects if images are imported
+        const problems = (response.length === 2)
+            ? response[1].problems
+            : response[0].problems;
+
+        frame.response = {
+            db: [],
+            problems: problems
+        };
+    },
+
+    deleteAllContent(response, apiConfig, frame) {
```

Comment:
Even though `deleteAllContent` responds with `statusCode: 204` this transform is needed in case the method is used internally.